Content.Client/GameObjects/Components/Mobs/ClientActionsComponent.cs

```diff
@@ -217,6 +217,15 @@ public bool TargetingOnUse(in PointerInputCmdHandler.PointerInputCmdArgs args)
                     }
                     return true;
                 }
+                // we're targeting entities, but the user didn't point at one
+                case BehaviorType.TargetEntity when args.EntityUid == EntityUid.Invalid:
+
+                    // If we don't repeat, stop targeting.
+                    if (!attempt.Action.Repeat)
```

Comment:
Shouldn't even non-repeating entity targeting actions do this? If you're trying to click an entity in the heat of the moment and you miss it would be annoying to have to re-select the action, regardless of if it's repeating or not.